events/wc_draenei_events.txt

```diff
@@ -0,0 +1,277 @@
+namespace = WCDRN
+
+############################
+#	Crashlanding of Exodar
+############################
+
+# Creates the Draenei and characters
+character_event = {
+	id = WCDRN.99
```

Comment:
This event slows down the game very much. It tries to fire EVERY month for EVERY character. I'll redesign it.